src/lib/urn.ts

```diff
@@ -92,6 +100,11 @@ export function toLegacyURN(urn: URN): URN {
   return urn.replace('urn:decentraland:off-chain:base-avatars:', 'dcl://base-avatars/')
 }
 
+export function getThirdPartyId(urn: URN): string {
```

Comment:
I'm not sure about the name of this method. I think the name can be a bit misleading because it's receiving an URN and returning the thirdPartyId, but it actually it's getting a third party URN and trying to extract the id from there.

Furthermore, the implementation could be reduced to:

```ts
decodeURN(collection.urn).suffix
```

Maybe we can just add a selector that does (and memoizes) this?

`getCollectionThirdParty(state, collection)` ?

Reply:
Renamed the `getThirdPartyId` method to `extractThirdPartyId`.